openpype/hosts/maya/api/lib.py

```diff
@@ -172,6 +174,218 @@ def maintained_selection():
             cmds.select(clear=True)
 
 
+def reload_all_udim_tile_previews():
+    """Regenerate all UDIM tile preview in texture file
+    nodes during context
+    """
+    texture_files = cmds.ls(type="file")
+    if texture_files:
+        for texture_file in texture_files:
+            if cmds.getAttr("{}.uvTilingMode".format(texture_file)) > 0:
+                cmds.ogs(regenerateUVTilePreview=texture_file)
+    cmds.ogs(reloadTextures=True)
+
+
+@contextlib.contextmanager
+def panel_camera(panel, camera):
+    original_camera = cmds.modelPanel(panel, query=True, camera=True)
+    try:
+        cmds.modelPanel(panel, edit=True, camera=camera)
+        yield
+    finally:
+        cmds.modelPanel(panel, edit=True, camera=original_camera)
+
+
+def playblast_capture(preset, instance):
+    """Function for playblast capturing with the preset options
+
+    Args:
+        preset (dict): preset options
+        instance (str): instance
+
+    Returns:
+        _type_: _description_
+    """
```

Comment:
```suggestion
def render_capture_preset(preset):
    """Capture playblast with a preset.
    
    To generate the preset use `generate_capture_preset`.

    Args:
        preset (dict): preset options
        instance (str): instance

    Returns:
        str: Output path of the playblast capture call
    """
```

Just make sure to set the `panel` already in the `generate_capture_preset` function from the instance data into the preset.
Also, replace `instance` as argument to `viewport_default_options` with `preset` - the preset would then contain the `panel` and there's no need for `instance` as argument anymore.

Reply:
Fixed in #6081